cmd/dowload_test.go

```diff
@@ -2,44 +2,37 @@ package cmd
 
 import (
 	"fmt"
-	"net/http"
-	"os"
 	"testing"
 
 	"github.com/cisco-open/grabit/test"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestRunDownload(t *testing.T) {
-	handler := func(w http.ResponseWriter, r *http.Request) {
-		_, err := w.Write([]byte(`abcdef`))
-		if err != nil {
-			t.Fatal(err)
-		}
-	}
-	port, server := test.HttpHandler(handler)
-	defer server.Close()
+	content := `abcdef`
+	port := test.TestHttpHandler(content, t)
 	testfilepath := test.TmpFile(t, fmt.Sprintf(`
 	[[Resource]]
 	Urls = ['http://localhost:%d/test.html']
 	Integrity = 'sha256-vvV+x/U6bUC+tkCngKY5yDvCmsipgW8fxsXG3Nk8RyE='
-`, port))
+
+	[[Resource]]
+	Urls = ['http://localhost:%d/test3.html']
+	Integrity = 'sha256-vvV+x/U6bUC+tkCngKY5yDvCmsipgW8fxsXG3Nk8RyE='
```

Comment:
A very minor point: Is there a way to not have to hard-code this sha256 sum?  E.g., could you compute it dynamically from the `content`?  This is obviously not a *problem* -- just might make it a little easier to have arbitrary content in the tests (vs. the human having to generate the content and matching sha256 sum).